### ISSUE TITLE
Can't load Ads and event callback not showing anything (Android)

### ISSUE THREAD

**Issue by zukinaru** [sakebook/flutter_native_ads#23]
Hi,

I place native ads widget on my `Home Screen`   and the structure like below:

For Initialization:
```
class HomeScreen extends StatefulWidget{
  @override
  _HomeScreenState createState() => _HomeScreenState();
}

class _HomeScreenState extends State<HomeScreen> with AfterLayoutMixin<HomeScreen> {

  @override
    void initState() {
      // TODO: implement initState
      super.initState();      
      NativeAds.initialize();
      ...
    }
```

For **NativeAds** placement is under below structures:
```
@override
Widget build(BuildContext context) {
  return Stack(
    children: <Widget>[
      Scaffold(
        appBar: AppBar( .. ),
        body: OfflineBuilder(
              connectivityBuilder: (BuildContext context, ConnectivityResult connectivity, Widget child) {
                  ...  
                  child: NestedScrollView(
                      headerSliverBuilder: (BuildContext context, bool innerBoxIsScrolled) { ... }
                      body: RefreshIndicator(
                        backgroundColor: Pigment.fromString(UIData.primaryColor),
                        color: Colors.white,
                        onRefresh: () => homeBloc.init(),
                        child: ListView(
                          children: <Widget>[
                            Row(
                            ...
                              ListView(
                                physics: NeverScrollableScrollPhysics(),
                                shrinkWrap: true,
                                children: <Widget>[
                                 
                                ...

                                  
                                SizedBox(
                                  width: double.infinity,
                                  height: 320,
                                  child: NativeAdView(
                                    onParentViewCreated: (_) {
                                      print("sinative create.....");
                                    },
                                    androidParam: AndroidParam()
                                      ..placementId = "ca-app-pub-xxx" //my admob
                                      ..packageName = "com.apps.myapp"
                                      ..layoutName = "native_ad_layout"
                                      ..attributionText = "AD",
                                    iosParam: IOSParam()
                                      ..placementId = "ca-app-pub-3940256099942544/3986624511" // test
                                      ..bundleId = "{{YOUR_IOS_APP_BUNDLE_ID}}"
                                      ..layoutName = "{{YOUR_CREATED_LAYOUT_FILE_NAME}}"
                                      ..attributionText = "SPONSORED",
                                    onAdImpression: () => print("sinative_muncul"),
                                    onAdClicked: () => print("onAdClicked!!!"),
                                    onAdFailedToLoad: (Map<String, dynamic> error) => print("sinative_error ==> $error"),
                                  ),
                                ),                      
                                
                                ...

                                ...


```

However, **Native Ads** wont load and show, also the event callback like `onAdFailedToLoad` does not return anything...

any idea to solve ?

Thank You in advance...

**Comment by zukinaru** [comment]
こんにちは @sakebook 

Do you have any idea ?

Thank You

**Comment by sakebook** [comment]
@zukinaru 

Probably your implementation is wrong.

`NativeAds.initialize()` call only once.

And have you created a layout file?

Please error log.

**Comment by zukinaru** [comment]
@sakebook 

NativeAds.initialize() only call once on `main.dart`:

FlutterLocalNotificationsPlugin flutterLocalNotificationsPlugin;

```
void main() async {
  await translations.init();
  print(Security.getSignature());
  SystemChrome.setSystemUIOverlayStyle(SystemUiOverlayStyle(
    statusBarColor: Pigment.fromString(UIData.primaryColor),
  ));

  SystemChrome.setPreferredOrientations([DeviceOrientation.portraitUp])
    .then((_) {
    registerDependencies();

    NativeAds.initialize();

    runApp(new AlertProvider(
        child: new App(),
        config: new AlertConfig(ok: "OK", cancel: "CANCEL"),
      ));
    });
}
```

and yes, i have created the layout like below:

(in android/app/src/main/res/layout/native_ad_layout.xml)

```
<?xml version="1.0" encoding="utf-8"?>
<com.google.android.gms.ads.formats.UnifiedNativeAdView
    xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:tools="http://schemas.android.com/tools"
    android:id="@+id/flutter_native_ad_unified_native_ad"
    android:layout_width="match_parent"
    android:layout_height="wrap_content">

    <RelativeLayout
        android:layout_width="match_parent"
        android:layout_height="wrap_content">

        <TextView
            android:id="@+id/flutter_native_ad_headline"
            android:layout_width="match_parent"
            android:layout_height="wrap_content"
            android:layout_alignParentLeft="true"
            android:layout_alignParentTop="true"
            android:layout_alignParentRight="true"
            android:layout_marginLeft="12dp"
            android:layout_marginRight="12dp"
            android:ellipsize="end"
            android:maxLines="2"
            android:textColor="@android:color/black"
            android:textSize="14sp"
            android:textStyle="bold"
            tools:ignore="RtlHardcoded" />

        <TextView
            android:id="@+id/flutter_native_ad_body"
            android:layout_width="match_parent"
            android:layout_height="wrap_content"
            android:layout_below="@+id/flutter_native_ad_headline"
            android:layout_alignParentLeft="true"
            android:layout_alignParentRight="true"
            android:layout_marginLeft="12dp"
            android:layout_marginTop="12dp"
            android:layout_marginRight="12dp"
            android:ellipsize="end"
            android:maxLines="2"
            android:textColor="@android:color/black"
            android:textSize="14sp"
            tools:ignore="RtlHardcoded"
            tools:text="sample title. sample title. sample title. sample title. sample title. sample title" />

        <com.google.android.gms.ads.formats.MediaView
            android:id="@+id/flutter_native_ad_media"
            android:layout_width="match_parent"
            android:layout_height="220dp"
            android:layout_below="@+id/flutter_native_ad_body"
            android:layout_alignParentStart="true"
            android:layout_alignParentLeft="true"
            android:layout_alignParentEnd="true"
            android:layout_alignParentRight="true"
            android:layout_marginLeft="12dp"
            android:layout_marginRight="12dp" />

        <TextView
            android:id="@+id/flutter_native_ad_attribution"
            android:layout_width="wrap_content"
            android:layout_height="24dp"
            android:layout_alignParentStart="true"
            android:layout_alignParentLeft="true"
            android:layout_alignParentBottom="true"
            android:layout_marginStart="12dp"
            android:layout_marginLeft="12dp"
            android:ellipsize="middle"
            android:gravity="center"
            android:singleLine="true"
            android:textColor="@android:color/darker_gray"
            android:textSize="12sp" />

        <TextView
            android:id="@+id/flutter_native_ad_call_to_action"
            android:layout_width="wrap_content"
            android:layout_height="24dp"
            android:layout_alignParentEnd="true"
            android:layout_alignParentRight="true"
            android:layout_alignParentBottom="true"
            android:layout_marginEnd="12dp"
            android:layout_marginRight="12dp"
            android:background="@drawable/rounded_corner"
            android:ellipsize="middle"
            android:gravity="center"
            android:includeFontPadding="false"
            android:singleLine="true"
            android:textColor="@android:color/white"
            android:textSize="12sp"
            android:textStyle="bold" />

    </RelativeLayout>

</com.google.android.gms.ads.formats.UnifiedNativeAdView>
```

any idea to solve ?

**Comment by sakebook** [comment]
I want to know the error log.

Probably there is a problem other than getting ads.

**Comment by zukinaru** [comment]
Hi @sakebook 
Here're the full log:

```
Running "flutter pub get" in ads_upgrade...                         1.9s
Launching lib/main.dart on SM T715Y in debug mode...
Initializing gradle...
Resolving dependencies...
Running Gradle task 'assembleDebug'...
registerResGeneratingTask is deprecated, use registerGeneratedResFolders(FileCollection)
registerResGeneratingTask is deprecated, use registerGeneratedResFolders(FileCollection)
registerResGeneratingTask is deprecated, use registerGeneratedResFolders(FileCollection)
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Built build/app/outputs/apk/debug/app-debug.apk.
Installing build/app/outputs/apk/app.apk...
V/InputMethodManager( 6086): Starting input: tba=android.view.inputmethod.EditorInfo@b5bdc19 nm : com.ptcreative.appreader ic=null
I/TAG     ( 6086): All location settings are satisfied.
I/flutter ( 6086): QVBQUy0xIDE1NzAwNjAzMzgyNTQ=
I/System.out( 6086): (HTTPLog)-Static: isSBSettingEnabled false
I/System.out( 6086): (HTTPLog)-Static: isSBSettingEnabled false
I/art     ( 6086): Background partial concurrent mark sweep GC freed 18804(1243KB) AllocSpace objects, 13(568KB) LOS objects, 39% free, 6MB/10MB, paused 1.089ms total 104.309ms
I/System.out( 6086): (HTTPLog)-Static: isSBSettingEnabled false
I/System.out( 6086): (HTTPLog)-Static: isSBSettingEnabled false
Syncing files to device SM T715Y...
D/ViewRootImpl@1d46018[MainActivity]( 6086): MSG_WINDOW_FOCUS_CHANGED 0
I/FIAM.Headless( 6086): Removing display event listener
D/SensorManager( 6086): unregisterListener ::   
I/System.out( 6086): (HTTPLog)-Static: isSBSettingEnabled false
I/System.out( 6086): (HTTPLog)-Static: isSBSettingEnabled false
D/InputTransport( 6086): Input channel destroyed: fd=120
I/flutter ( 6086): get Foundation
I/flutter ( 6086): get Greetings
I/flutter ( 6086): get master locations
E/flutter ( 6086): [ERROR:flutter/lib/ui/ui_dart_state.cc(148)] Unhandled Exception: MissingPluginException(No implementation found for method initialize on channel native_ads)
E/flutter ( 6086): #0      MethodChannel.invokeMethod (package:flutter/src/services/platform_channel.dart:314:7)
E/flutter ( 6086): <asynchronous suspension>
E/flutter ( 6086): #1      new NativeAds.initialize (package:native_ads/native_ads.dart:5:14)
E/flutter ( 6086): #2      main.<anonymous closure> (package:appreader/main.dart:35:15)
E/flutter ( 6086): #3      _rootRunUnary (dart:async/zone.dart:1132:38)
E/flutter ( 6086): #4      _CustomZone.runUnary (dart:async/zone.dart:1029:19)
E/flutter ( 6086): #5      _FutureListener.handleValue (dart:async/future_impl.dart:137:18)
E/flutter ( 6086): #6      Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:678:45)
E/flutter ( 6086): #7      Future._propagateToListeners (dart:async/future_impl.dart:707:32)
E/flutter ( 6086): #8      Future._completeWithValue (dart:async/future_impl.dart:522:5)
E/flutter ( 6086): #9      _AsyncAwaitCompleter.complete (dart:async-patch/async_patch.dart:30:15)
E/flutter ( 6086): #10     _completeOnAsyncReturn (dart:async-patch/async_patch.dart:288:13)
E/flutter ( 6086): #11     SystemChrome.setPreferredOrientations (package:flutter/src/services/system_chrome.dart)
E/flutter ( 6086): <asynchronous suspension>
E/flutter ( 6086): #12     main (package:appreader/main.dart:31:16)
E/flutter ( 6086): <asynchronous suspension>
E/flutter ( 6086): #13     _runMainZoned.<anonymous closure>.<anonymous closure> (dart:ui/hooks.dart:229:25)
E/flutter ( 6086): #14     _rootRun (dart:async/zone.dart:1124:13)
E/flutter ( 6086): #15     _CustomZone.run (dart:async/zone.dart:1021:19)
E/flutter ( 6086): #16     _runZoned (dart:async/zone.dart:1516:10)
E/flutter ( 6086): #17     runZoned (dart:async/zone.dart:1500:12)
E/flutter ( 6086): #18     _runMainZoned.<anonymous closure> (dart:ui/hooks.dart:221:5)
E/flutter ( 6086): #19     _startIsolate.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:305:19)
E/flutter ( 6086): #20     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:172:12)
I/flutter ( 6086): NoSuchMethodError: The getter 'length' was called on null.
I/flutter ( 6086): Receiver: null
I/flutter ( 6086): Tried calling: length
I/flutter ( 6086): NoSuchMethodError: The getter 'length' was called on null.
I/flutter ( 6086): Receiver: null
I/flutter ( 6086): Tried calling: length
I/flutter ( 6086): null
E/flutter ( 6086): [ERROR:flutter/lib/ui/ui_dart_state.cc(148)] Unhandled Exception: NoSuchMethodError: The getter 'length' was called on null.
E/flutter ( 6086): Receiver: null
E/flutter ( 6086): Tried calling: length
E/flutter ( 6086): #0      Object.noSuchMethod (dart:core-patch/object_patch.dart:51:5)
E/flutter ( 6086): #1      SplashBloc.getGreetings (package:appreader/src/blocs/greetings/splash_bloc.dart:123:37)
E/flutter ( 6086): <asynchronous suspension>
E/flutter ( 6086): #2      _SplashScreenState.init (package:appreader/src/ui/greeting/splash_screen.dart:46:18)
E/flutter ( 6086): #3      _asyncThenWrapperHelper.<anonymous closure> (dart:async-patch/async_patch.dart:71:64)
E/flutter ( 6086): #4      _rootRunUnary (dart:async/zone.dart:1132:38)
E/flutter ( 6086): #5      _CustomZone.runUnary (dart:async/zone.dart:1029:19)
E/flutter ( 6086): #6      _FutureListener.handleValue (dart:async/future_impl.dart:137:18)
E/flutter ( 6086): #7      Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:678:45)
E/flutter ( 6086): #8      Future._propagateToListeners (dart:async/future_impl.dart:707:32)
E/flutter ( 6086): #9      Future._completeWithValue (dart:async/future_impl.dart:522:5)
E/flutter ( 6086): #10     _AsyncAwaitCompleter.complete (dart:async-patch/async_patch.dart:30:15)
E/flutter ( 6086): #11     _completeOnAsyncReturn (dart:async-patch/async_patch.dart:288:13)
E/flutter ( 6086): #12     Session.getIsFirst (package:appreader/src/resources/session.dart)
E/flutter ( 6086): #13     _asyncThenWrapperHelper.<anonymous closure> (dart:async-patch/async_patch.dart:71:64)
E/flutter ( 6086): #14     _rootRunUnary (dart:async/zone.dart:1132:38)
E/flutter ( 6086): #15     _CustomZone.runUnary (dart:async/zone.dart:1029:19)
E/flutter ( 6086): #16     _FutureListener.handleValue (dart:async/future_impl.dart:137:18)
E/flutter ( 6086): #17     Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:678:45)
E/flutter ( 6086): #18     Future._propagateToListeners (dart:async/future_impl.dart:707:32)
E/flutter ( 6086): #19     Future._completeWithValue (dart:async/future_impl.dart:522:5)
E/flutter ( 6086): #20     Future._asyncComplete.<anonymous closure> (dart:async/future_impl.dart:552:7)
E/flutter ( 6086): #21     _rootRun (dart:async/zone.dart:1124:13)
E/flutter ( 6086): #22     _CustomZone.run (dart:async/zone.dart:1021:19)
E/flutter ( 6086): #23     _CustomZone.runGuarded (dart:async/zone.dart:923:7)
E/flutter ( 6086): #24     _CustomZone.bindCallbackGuarded.<anonymous closure> (dart:async/zone.dart:963:23)
E/flutter ( 6086): #25     _microtaskLoop (dart:async/schedule_microtask.dart:41:21)
E/flutter ( 6086): #26     _startMicrotaskLoop (dart:async/schedule_microtask.dart:50:5)
E/flutter ( 6086): 
E/FlutterFcmService( 6086): Fatal: failed to find callback
I/System.out( 6086): (HTTPLog)-Static: isSBSettingEnabled false
I/System.out( 6086): (HTTPLog)-Static: isSBSettingEnabled false
I/System.out( 6086): (HTTPLog)-Static: isSBSettingEnabled false
I/System.out( 6086): (HTTPLog)-Static: isSBSettingEnabled false
I/FIAM.Headless( 6086): went foreground
I/FIAM.Headless( 6086): Forcing fetch from service rather than cache. Test Device: false | App Fresh Install: true
W/FIAM.Headless( 6086): Recoverable exception while reading cache: /data/user/0/com.ptcreative.appreader/files/fiam_impressions_store_file (No such file or directory)
I/System.out( 6086): (HTTPLog)-Static: isSBSettingEnabled false
I/System.out( 6086): (HTTPLog)-Static: isSBSettingEnabled false
I/FIAM.Headless( 6086): Fetching campaigns from service.
W/DynamiteModule( 6086): Local module descriptor class for providerinstaller not found.
D/ViewRootImpl@1d46018[MainActivity]( 6086): MSG_WINDOW_FOCUS_CHANGED 1
D/ViewRootImpl@1d46018[MainActivity]( 6086): mHardwareRenderer.initializeIfNeeded()#2 mSurface={isValid=true 1813508096}
V/InputMethodManager( 6086): Starting input: tba=android.view.inputmethod.EditorInfo@aa1145f nm : com.ptcreative.appreader ic=null
I/InputMethodManager( 6086): [IMM] startInputInner - mService.startInputOrWindowGainedFocus
D/InputTransport( 6086): Input channel constructed: fd=65
I/DynamiteModule( 6086): Considering local module providerinstaller:0 and remote module providerinstaller:0
D/SensorManager( 6086): registerListener :: 1, LSM6DS3 Accelerometer Sensor, 66667, 0,  
W/ProviderInstaller( 6086): Failed to load providerinstaller module: No acceptable module found. Local version is 0 and remote version is 0.
V/NativeCrypto( 6086): Registering com/google/android/gms/org/conscrypt/NativeCrypto's 284 native methods...
I/ProviderInstaller( 6086): Installed default security provider GmsCore_OpenSSL
W/art     ( 6086): Before Android 4.1, method double java.util.concurrent.ThreadLocalRandom.internalNextDouble(double, double) would have incorrectly overridden the package-private method in java.util.Random
W/art     ( 6086): Before Android 4.1, method int java.util.concurrent.ThreadLocalRandom.internalNextInt(int, int) would have incorrectly overridden the package-private method in java.util.Random
W/art     ( 6086): Before Android 4.1, method long java.util.concurrent.ThreadLocalRandom.internalNextLong(long, long) would have incorrectly overridden the package-private method in java.util.Random
I/FIAM.Headless( 6086): Successfully fetched 0 messages from backend
D/ViewRootImpl@1d46018[MainActivity]( 6086): ViewPostImeInputStage processPointer 0
W/System  ( 6086): ClassLoader referenced unknown path: /system/framework/QPerformance.jar
E/BoostFramework( 6086): BoostFramework() : Exception_1 = java.lang.ClassNotFoundException: Didn't find class "com.qualcomm.qti.Performance" on path: DexPathList[[],nativeLibraryDirectories=[/system/lib, /vendor/lib]]
V/BoostFramework( 6086): BoostFramework() : mPerf = null
D/ViewRootImpl@1d46018[MainActivity]( 6086): ViewPostImeInputStage processPointer 1
I/art     ( 6086): Do partial code cache collection, code=23KB, data=30KB
I/art     ( 6086): After code cache collection, code=22KB, data=29KB
I/art     ( 6086): Increasing code cache capacity to 128KB
I/flutter ( 6086): NoSuchMethodError: The getter 'length' was called on null.
I/flutter ( 6086): Receiver: null
I/flutter ( 6086): Tried calling: length
I/flutter ( 6086): load from api
D/ViewRootImpl@1d46018[MainActivity]( 6086): ViewPostImeInputStage processPointer 0
D/ViewRootImpl@1d46018[MainActivity]( 6086): ViewPostImeInputStage processPointer 1
I/flutter ( 6086): Service status: true
D/ViewRootImpl@1d46018[MainActivity]( 6086): MSG_WINDOW_FOCUS_CHANGED 0
I/FIAM.Headless( 6086): Removing display event listener
D/SensorManager( 6086): unregisterListener ::   
D/ViewRootImpl@1d46018[MainActivity]( 6086): MSG_WINDOW_FOCUS_CHANGED 1
D/ViewRootImpl@1d46018[MainActivity]( 6086): mHardwareRenderer.initializeIfNeeded()#2 mSurface={isValid=true 1813508096}
I/flutter ( 6086): Permission: true
D/SensorManager( 6086): registerListener :: 1, LSM6DS3 Accelerometer Sensor, 66667, 0,  
D/ViewRootImpl@1d46018[MainActivity]( 6086): Relayout returned: oldFrame=[0,0][1536,2048] newFrame=[0,0][1536,2048] result=0x1 surface={isValid=true 1813508096} surfaceGenerationChanged=false
D/ViewRootImpl@1d46018[MainActivity]( 6086): ViewPostImeInputStage processPointer 0
D/ViewRootImpl@1d46018[MainActivity]( 6086): ViewPostImeInputStage processPointer 1
I/art     ( 6086): Do partial code cache collection, code=42KB, data=62KB
I/art     ( 6086): After code cache collection, code=42KB, data=62KB
I/art     ( 6086): Increasing code cache capacity to 256KB 
D/ViewRootImpl@1d46018[MainActivity]( 6086): ViewPostImeInputStage processPointer 0
D/ViewRootImpl@1d46018[MainActivity]( 6086): ViewPostImeInputStage processPointer 1
E/flutter ( 6086): [ERROR:flutter/lib/ui/ui_dart_state.cc(148)] Unhandled Exception: NoSuchMethodError: The method 'dispose' was called on null.
E/flutter ( 6086): Receiver: null
E/flutter ( 6086): Tried calling: dispose()
E/flutter ( 6086): #0      Object.noSuchMethod (dart:core-patch/object_patch.dart:51:5)
E/flutter ( 6086): #1      AdsHelper.hideBannerAd (package:appreader/src/helpers/ads_helper.dart:65:21)
E/flutter ( 6086): <asynchronous suspension>
E/flutter ( 6086): #2      _LoginScreenState.hideBanner (package:appreader/src/ui/auth/login_screen.dart:56:15)
E/flutter ( 6086): <asynchronous suspension>
E/flutter ( 6086): #3      _LoginScreenState.initState (package:appreader/src/ui/auth/login_screen.dart:31:7)
E/flutter ( 6086): #4      StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:4068:58)
E/flutter ( 6086): #5      ComponentElement.mount (package:flutter/src/widgets/framework.dart:3919:5)
E/flutter ( 6086): #6      Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
E/flutter ( 6086): #7      Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
E/flutter ( 6086): #8      SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:5127:14)
E/flutter ( 6086): #9      Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
E/flutter ( 6086): #10     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
E/flutter ( 6086): #11     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3961:16)
E/flutter ( 6086): #12     Element.rebuild (package:flutter/src/widgets/framework.dart:3738:5)
E/flutter ( 6086): #13     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3924:5)
E/flutter ( 6086): #14     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3919:5)
E/flutter ( 6086): #15     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
E/flutter ( 6086): #16     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
E/flutter ( 6086): #17     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:5127:14)
E/flutter ( 6086): #18     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
E/flutter ( 6086): #19     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
E/flutter ( 6086): #20     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:5127:14)
E/flutter ( 6086): #21     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
E/flutter ( 6086): #22     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
E/flutter ( 6086): #23     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:5127:14)
E/flutter ( 6086): #24     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
E/flutter ( 6086): #25     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
E/flutter ( 6086): #26     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:5127:14)
E/flutter ( 6086): #27     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
E/flutter ( 6086): #28     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
E/flutter ( 6086): #29     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3961:16)
E/flutter ( 6086): #30     Element.rebuild (package:flutter/src/widgets/framework.dart:3738:5)
E/flutter ( 6086): #31     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3924:5)
E/flutter ( 6086): #32     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:4088:11)
E/flutter ( 6086): #33     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3919:5)
E/flutter ( 6086): #34     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
E/flutter ( 6086): #35     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
E/flutter ( 6086): #36     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3961:16)
E/flutter ( 6086): #37     Element.rebuild (package:flutter/src/widgets/framework.dart:3738:5)
E/flutter ( 6086): #38     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3924:5)
E/flutter ( 6086): #39     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3919:5)
E/flutter ( 6086): #40     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
E/flutter ( 6086): #41     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
E/flutter ( 6086): #42     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3961:16)
E/flutter ( 6086): #43     Element.rebuild (package:flutter/src/widgets/framework.dart:3738:5)
E/flutter ( 6086): #44     ComponentElement._firstBuild (package:flutter/src/widgets/framework.d
D/ViewRootImpl@1d46018[MainActivity]( 6086): ViewPostImeInputStage processPointer 0
D/ViewRootImpl@1d46018[MainActivity]( 6086): ViewPostImeInputStage processPointer 1
I/flutter ( 6086): "skip login
I/flutter ( 6086): AsyncSnapshot<String>(ConnectionState.waiting, null, null)
I/flutter ( 6086): Service status: true
D/ViewRootImpl@1d46018[MainActivity]( 6086): MSG_WINDOW_FOCUS_CHANGED 0
I/FIAM.Headless( 6086): Removing display event listener
D/SensorManager( 6086): unregisterListener ::   
D/ViewRootImpl@1d46018[MainActivity]( 6086): MSG_WINDOW_FOCUS_CHANGED 1
D/ViewRootImpl@1d46018[MainActivity]( 6086): mHardwareRenderer.initializeIfNeeded()#2 mSurface={isValid=true 1813508096}
I/flutter ( 6086): AsyncSnapshot<String>(ConnectionState.active, Thursday, October 3, 2019, null)
D/SensorManager( 6086): registerListener :: 1, LSM6DS3 Accelerometer Sensor, 66667, 0,  
I/flutter ( 6086): 0
D/ViewRootImpl@1d46018[MainActivity]( 6086): Relayout returned: oldFrame=[0,0][1536,2048] newFrame=[0,0][1536,2048] result=0x1 surface={isValid=true 1813508096} surfaceGenerationChanged=false
I/flutter ( 6086): Permission: true
E/flutter ( 6086): [ERROR:flutter/lib/ui/ui_dart_state.cc(148)] Unhandled Exception: PlatformException(error, Trying to create a platform view of unregistered type: com.github.sakebook.android/unified_ad_layout, null)
E/flutter ( 6086): #0      StandardMethodCodec.decodeEnvelope (package:flutter/src/services/message_codecs.dart:569:7)
E/flutter ( 6086): #1      MethodChannel.invokeMethod (package:flutter/src/services/platform_channel.dart:316:33)
E/flutter ( 6086): <asynchronous suspension>
E/flutter ( 6086): #2      AndroidViewController._create (package:flutter/src/services/platform_views.dart:640:54)
E/flutter ( 6086): <asynchronous suspension>
E/flutter ( 6086): #3      AndroidViewController.setSize (package:flutter/src/services/platform_views.dart:557:14)
E/flutter ( 6086): <asynchronous suspension>
E/flutter ( 6086): #4      RenderAndroidView._sizePlatformView (package:flutter/src/rendering/platform_view.dart:174:29)
E/flutter ( 6086): <asynchronous suspension>
E/flutter ( 6086): #5      RenderAndroidView.performResize (package:flutter/src/rendering/platform_view.dart:155:5)
E/flutter ( 6086): #6      RenderObject.layout (package:flutter/src/rendering/object.dart:1686:9)
E/flutter ( 6086): #7      RenderConstrainedBox.performLayout (package:flutter/src/rendering/proxy_box.dart:259:13)
E/flutter ( 6086): #8      RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
E/flutter ( 6086): #9      RenderFlex.performLayout (package:flutter/src/rendering/flex.dart:744:15)
E/flutter ( 6086): #10     RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
E/flutter ( 6086): #11     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
E/flutter ( 6086): #12     RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
E/flutter ( 6086): #13     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
E/flutter ( 6086): #14     RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
E/flutter ( 6086): #15     RenderSliverMultiBoxAdaptor.insertAndLayoutChild (package:flutter/src/rendering/sliver_multi_box_adaptor.dart:492:13)
E/flutter ( 6086): #16     RenderSliverList.performLayout.advance (package:flutter/src/rendering/sliver_list.dart:195:19)
E/flutter ( 6086): #17     RenderSliverList.performLayout (package:flutter/src/rendering/sliver_list.dart:238:19)
E/flutter ( 6086): #18     RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
E/flutter ( 6086): #19     RenderSliverPadding.performLayout (package:flutter/src/rendering/sliver_padding.dart:181:11)
E/flutter ( 6086): #20     RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
E/flutter ( 6086): #21     RenderViewportBase.layoutChildSequence (package:flutter/src/rendering/viewport.dart:406:13)
E/flutter ( 6086): #22     RenderShrinkWrappingViewport._attemptLayout (package:flutter/src/rendering/viewport.dart:1681:12)
E/flutter ( 6086): #23     RenderShrinkWrappingViewport.performLayout (package:flutter/src/rendering/viewport.dart:1644:20)
E/flutter ( 6086): #24     RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
E/flutter ( 6086): #25     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
E/flutter ( 6086): #26     RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
E/flutter ( 6086): #27     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
E/flutter ( 6086): #28     RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
E/flutter ( 6086): #29     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
E/flutter ( 6086): #30     RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
E/flutter ( 6086): #31     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
E/flutter ( 6086): #32     RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
E/flutter ( 6086): #33     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
E/flutter ( 6086): #34     RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
E/flutter ( 6086): #35     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
E/flutter ( 6086): #36     RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
E/flutter ( 6086): #37     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
E/flutter ( 6086): #38     RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
E/flutter ( 6086): #39     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
E/flutter ( 6086): #40     RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
E/flutter ( 6086): #41     RenderProxyBoxMixin.pe
I/flutter ( 6086): NoSuchMethodError: The method '+' was called on null.
I/flutter ( 6086): Receiver: null
I/flutter ( 6086): Tried calling: +(", ")
I/DynamiteModule( 6086): Considering local module com.google.android.gms.ads.dynamite:0 and remote module com.google.android.gms.ads.dynamite:20461
I/DynamiteModule( 6086): Selected remote version of com.google.android.gms.ads.dynamite, version >= 20461
V/DynamiteModule( 6086): Dynamite loader version >= 2, using loadModule2NoCrashUtils
W/ResourceType( 6086): Found multiple library tables, ignoring...
W/ResourceType( 6086): Found multiple library tables, ignoring...
I/WebViewFactory( 6086): Loading com.android.chrome version 77.0.3865.92 (code 386509207)
W/art     ( 6086): Suspending all threads took: 14.123ms
I/art     ( 6086): Background sticky concurrent mark sweep GC freed 34426(1735KB) AllocSpace objects, 6(120KB) LOS objects, 13% free, 8MB/10MB, paused 24.038ms total 506.786ms
I/art     ( 6086): Rejecting re-init on previously-failed class java.lang.Class<Mk>: java.lang.NoClassDefFoundError: Failed resolution of: Landroid/webkit/TracingController;
I/art     ( 6086):   at java.lang.Class java.lang.Class.classForName!(java.lang.String, boolean, java.lang.ClassLoader) (Class.java:-2)
I/art     ( 6086):   at java.lang.Class java.lang.Class.forName(java.lang.String, boolean, java.lang.ClassLoader) (Class.java:400)
I/art     ( 6086):   at java.lang.Class android.webkit.WebViewFactory.getProviderClass() (WebViewFactory.java:349)
I/art     ( 6086):   at android.webkit.WebViewFactoryProvider android.webkit.WebViewFactory.getProvider() (WebViewFactory.java:194)
I/art     ( 6086):   at android.webkit.CookieManager android.webkit.CookieManager.getInstance() (CookieManager.java:39)
I/art     ( 6086):   at android.webkit.CookieManager com.google.android.gms.internal.ads.zzaux.zzbd(android.content.Context) (com.google.android.gms:play-services-ads@@18.2.0:4)
I/art     ( 6086):   at java.lang.Object com.google.android.gms.internal.ads.zzcdd.call() (com.google.android.gms:play-services-ads@@18.2.0:3)
I/art     ( 6086):   at java.lang.Object com.google.android.gms.internal.ads.zzddz.zzapg() (com.google.android.gms:play-services-ads@@18.2.0:5)
I/art     ( 6086):   at void com.google.android.gms.internal.ads.zzddh.run() (com.google.android.gms:play-services-ads@@18.2.0:8)
I/art     ( 6086):   at void com.google.android.gms.internal.ads.zzddw.run() (com.google.android.gms:play-services-ads@@18.2.0:8)
I/art     ( 6086):   at void java.util.concurrent.ThreadPoolExecutor.runWorker(java.util.concurrent.ThreadPoolExecutor$Worker) (ThreadPoolExecutor.java:1133)
I/art     ( 6086):   at void java.util.concurrent.ThreadPoolExecutor$Worker.run() (ThreadPoolExecutor.java:607)
I/art     ( 6086):   at void java.lang.Thread.run() (Thread.java:762)
I/art     ( 6086): Caused by: java.lang.ClassNotFoundException: Didn't find class "android.webkit.TracingController" on path: DexPathList[[zip file "/data/app/com.android.chrome-2/base.apk", zip file "/data/app/com.android.chrome-2/split_autofill_assistant.apk", zip file "/data/app/com.android.chrome-2/split_autofill_assistant.config.en.apk", zip file "/data/app/com.android.chrome-2/split_autofill_assistant.config.in.apk", zip file "/data/app/com.android.chrome-2/split_config.en.apk", zip file "/data/app/com.android.chrome-2/split_config.in.apk"],nativeLibraryDirectories=[/data/app/com.android.chrome-2/lib/arm, /data/app/com.android.chrome-2/base.apk!/lib/armeabi-v7a, /data/app/com.android.chrome-2/split_autofill_assistant.apk!/lib/armeabi-v7a, /data/app/com.android.chrome-2/split_autofill_assistant.config.en.apk!/lib/armeabi-v7a, /data/app/com.android.chrome-2/split_autofill_assistant.config.in.apk!/lib/armeabi-v7a, /data/app/com.android.chrome-2/split_config.en.apk!/lib/armeabi-v7a, /data/app/com.android.chrome-2/split_confi
I/art     ( 6086):   at java.lang.Class dalvik.system.BaseDexClassLoader.findClass(java.lang.String) (BaseDexClassLoader.java:56)
I/art     ( 6086):   at java.lang.Class java.lang.ClassLoader.loadClass(java.lang.String, boolean) (ClassLoader.java:380)
I/art     ( 6086):   at java.lang.Class java.lang.ClassLoader.loadClass(java.lang.String) (ClassLoader.java:312)
I/art     ( 6086):   at java.lang.Class java.lang.Class.classForName!(java.lang.String, boolean, java.lang.ClassLoader) (Class.java:-2)
I/art     ( 6086):   at java.lang.Class java.lang.Class.forName(java.lang.String, boolean, java.lang.ClassLoader) (Class.java:400)
I/art     ( 6086):   at java.lang.Class android.webkit.WebViewFactory.getProviderClass() (WebViewFactory.java:349)
I/art     ( 6086):   at android.webkit.WebViewFactoryProvider android.webkit.WebViewFactory.getProvider() (WebViewFactory.java:194)
I/art     ( 6086):   at android.webkit.CookieManager android.webkit.CookieManager.getInstance() (CookieManager.java:39)
I/art     ( 6086):   at android.webkit.CookieManager com.google.android.gms.internal.ads.zzaux.zzbd(android.content.Context) (com.google.android.gms:play-services-ads@@18.2.0:4)
I/art     ( 6086):   at java.lang.Object com.google.android.gms.internal.ads.zzcdd.call() (com.google.android.gms:play-services-ads@@18.2.0:3)
I/art     ( 6086):   at java.lang.Object com.google.android.gms.internal.ads.zzddz.zzapg() (com.google.android.gms:play-services-ads@@18.2.0:5)
I/art     ( 6086):   at void com.google.android.gms.internal.ads.zzddh.run() (com.google.android.gms:play-services-ads@@18.2.0:8)
I/art     ( 6086):   at void com.google.android.gms.internal.ads.zzddw.run() (com.google.android.gms:play-services-ads@@18.2.0:8)
I/art     ( 6086):   at void java.util.concurrent.ThreadPoolExecutor.runWorker(java.util.concurrent.ThreadPoolExecutor$Worker) (ThreadPoolExecutor.java:1133)
I/art     ( 6086):   at void java.util.concurrent.ThreadPoolExecutor$Worker.run() (ThreadPoolExecutor.java:607)
I/art     ( 6086):   at void java.lang.Thread.run() (Thread.java:762)
I/art     ( 6086): 
I/cr_LibraryLoader( 6086): Time to load native libraries: 5 ms (timestamps 4429-4434)
I/chromium( 6086): [INFO:library_loader_hooks.cc(51)] Chromium logging enabled: level = 0, default verbosity = 0
I/cr_LibraryLoader( 6086): Expected native library version number "77.0.3865.92", actual native library version number "77.0.3865.92"
D/zzdx    ( 6086): File /data/user/0/com.ptcreative.appreader/cache/1561154238473.dex not found. No need for deletion
D/ViewRootImpl@1d46018[MainActivity]( 6086): ViewPostImeInputStage processPointer 0
D/ViewRootImpl@1d46018[MainActivity]( 6086): ViewPostImeInputStage processPointer 1
I/cr_BrowserStartup( 6086): Initializing chromium process, singleProcess=true
W/ResourceType( 6086): Failure getting entry for 0x7f13052f (t=18 e=1327) (error -2147483647)
E/chromium( 6086): [ERROR:filesystem_posix.cc(89)] stat /data/user/0/com.ptcreative.appreader/cache/WebView/Crashpad: No such file or directory (2)
E/chromium( 6086): [ERROR:filesystem_posix.cc(62)] mkdir /data/user/0/com.ptcreative.appreader/cache/WebView/Crashpad: No such file or directory (2)
I/art     ( 6086): Rejecting re-init on previously-failed class java.lang.Class<lk>: java.lang.NoClassDefFoundError: Failed resolution of: Landroid/webkit/SafeBrowsingResponse;
I/art     ( 6086):   at In com.android.webview.chromium.WebViewChromiumFactoryProvider.a(android.webkit.WebView, android.content.Context) (PG:9)
I/art     ( 6086):   at void com.android.webview.chromium.WebViewChromium.init(java.util.Map, boolean) (PG:23)
I/art     ( 6086):   at void android.webkit.WebView.<init>(android.content.Context, android.util.AttributeSet, int, int, java.util.Map, boolean) (WebView.java:636)
I/art     ( 6086):   at void android.webkit.WebView.<init>(android.content.Context, android.util.AttributeSet, int, int) (WebView.java:572)
I/art     ( 6086):   at void android.webkit.WebView.<init>(android.content.Context, android.util.AttributeSet, int) (WebView.java:555)
I/art     ( 6086):   at void android.webkit.WebView.<init>(android.content.Context, android.util.AttributeSet) (WebView.java:542)
I/art     ( 6086):   at void android.webkit.WebView.<init>(android.content.Context) (WebView.java:532)
I/art     ( 6086):   at void com.google.android.gms.internal.ads.zzbck.<init>(com.google.android.gms.internal.ads.zzbdk, com.google.android.gms.internal.ads.zzbdj, java.lang.String, boolean, boolean, com.google.android.gms.internal.ads.zzdf, com.google.android.gms.internal.ads.zzaxl, com.google.android.gms.internal.ads.zzaab, com.google.android.gms.ads.internal.zzi, com.google.android.gms.ads.internal.zza, com.google.android.gms.internal.ads.zzsd, com.google.android.gms.internal.ads.zzrf, boolean) (com.google.android.gms:play-services-ads@@18.2.0:3)
I/art     ( 6086):   at com.google.android.gms.internal.ads.zzbck com.google.android.gms.internal.ads.zzbck.zzb(android.content.Context, com.google.android.gms.internal.ads.zzbdj, java.lang.String, boolean, boolean, com.google.android.gms.internal.ads.zzdf, com.google.android.gms.internal.ads.zzaxl, com.google.android.gms.internal.ads.zzaab, com.google.android.gms.ads.internal.zzi, com.google.android.gms.ads.internal.zza, com.google.android.gms.internal.ads.zzsd, com.google.android.gms.internal.ads.zzrf, boolean) (com.google.android.gms:play-services-ads@@18.2.0:2)
I/art     ( 6086):   at java.lang.Object com.google.android.gms.internal.ads.zzbcd.call() (com.google.android.gms:play-services-ads@@18.2.0:3)
I/art     ( 6086):   at java.lang.Object com.google.android.gms.internal.ads.zzawq.zza(java.util.concurrent.Callable) (com.google.android.gms:play-services-ads-lite@@18.2.0:23)
I/art     ( 6086):   at com.google.android.gms.internal.ads.zzbbw com.google.android.gms.internal.ads.zzbcb.zza(android.content.Context, com.google.android.gms.internal.ads.zzbdj, java.lang.String, boolean, boolean, com.google.android.gms.internal.ads.zzdf, com.google.android.gms.internal.ads.zzaxl, com.google.android.gms.internal.ads.zzaab, com.google.android.gms.ads.internal.zzi, com.google.android.gms.ads.internal.zza, com.google.android.gms.internal.ads.zzsd, com.google.android.gms.internal.ads.zzrf, boolean) (com.google.android.gms:play-services-ads@@18.2.0:10)
I/art     ( 6086):   at void com.google.android.gms.internal.ads.zzahc.<init>(android.content.Context, com.google.android.gms.internal.ads.zzaxl, com.google.android.gms.internal.ads.zzdf, com.google.android.gms.ads.internal.zza) (com.google.android.gms:play-services-ads@@18.2.0:7)
I/art     ( 6086):   at void com.google.android.gms.internal.ads.zzahn.zza(com.google.android.gms.internal.ads.zzdf, com.google.android.gms.internal.ads.zzaie) (com.google.android.gms:play-services-ads@@18.2.0:54)
I/art     ( 6086):   at void com.google.android.gms.internal.ads.zzahm.run() (com.google.android.gms:play-services-ads@@18.2.0:-1)
I/art     ( 6086):   at void android.os.Handler.handleCallback(android.os.Message) (Handler.java:751)
I/art     ( 6086):   at void android.os.Handler.dispatchMessage(android.os.Message) (Handler.java:95)
I/art     ( 6086):   at void com.google.android.gms.internal.ads.zzdac.zzb(android.os.Message) (com.google.android.gms:play-services-ads-lite@@18.2.0:7)
I/art     ( 6086):   at void com.google.android.gms.internal.ads.zzauf.zzb(android.os.Message) (com.google.android.gms:play-services-ads@@18.2.0:8)
I/art     ( 6086):   at void com.google.android.gms.internal.ads.zzdac.dispatchMessage(android.os.Message) (com.google.android.gms:play-services-ads-lite@@18.2.0:5)
I/art     ( 6086):   at void android.os.Looper.loop() (Looper.java:154)
I/art     ( 6086):   at void android.app.ActivityThread.main(java.lang.String[]) (ActivityThread.java:6780)
I/art     ( 6086):   at java.lang.Object java.lang.reflect.Method.invoke!(java.lang.Object, java.lang.Object[]) (Method.java:-2)
I/art     ( 6086):   at void com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run() (ZygoteInit.java:1496)
I/art     ( 6086):   at void com.android.internal.os.ZygoteInit.main(java.lang.String[]) (ZygoteInit.java:1386)
I/art     ( 6086): Caused by: java.lang.ClassNotFoundException: Didn't find class "android.webkit.SafeBrowsingResponse" on path: DexPathList[[zip file "/data/app/com.android.chrome-2/base.apk", zip file "/data/app/com.android.chrome-2/split_autofill_assistant.apk", zip file "/data/app/com.android.chrome-2/split_autofill_assistant.config.en.apk", zip file "/data/app/com.android.chrome-2/split_autofill_assistant.config.in.apk", zip file "/data/app/com.android.chrome-2/split_config.en.apk", zip file "/data/app/com.android.chrome-2/split_config.in.apk"],nativeLibraryDirectories=[/data/app/com.android.chrome-2/lib/arm, /data/app/com.android.chrome-2/base.apk!/lib/armeabi-v7a, /data/app/com.android.chrome-2/split_autofill_assistant.apk!/lib/armeabi-v7a, /data/app/com.android.chrome-2/split_autofill_assistant.config.en.apk!/lib/armeabi-v7a, /data/app/com.android.chrome-2/split_autofill_assistant.config.in.apk!/lib/armeabi-v7a, /data/app/com.android.chrome-2/split_config.en.apk!/lib/armeabi-v7a, /data/app/com.android.chrome-2/split_co
I/art     ( 6086):   at java.lang.Class dalvik.system.BaseDexClassLoader.findClass(java.lang.String) (BaseDexClassLoader.java:56)
I/art     ( 6086):   at java.lang.Class java.lang.ClassLoader.loadClass(java.lang.String, boolean) (ClassLoader.java:380)
I/art     ( 6086):   at java.lang.Class java.lang.ClassLoader.loadClass(java.lang.String) (ClassLoader.java:312)
I/art     ( 6086):   at In com.android.webview.chromium.WebViewChromiumFactoryProvider.a(android.webkit.WebView, android.content.Context) (PG:9)
I/art     ( 6086):   at void com.android.webview.chromium.WebViewChromium.init(java.util.Map, boolean) (PG:23)
I/art     ( 6086):   at void android.webkit.WebView.<init>(android.content.Context, android.util.AttributeSet, int, int, java.util.Map, boolean) (WebView.java:636)
I/art     ( 6086):   at void android.webkit.WebView.<init>(android.content.Context, android.util.AttributeSet, int, int) (WebView.java:572)
I/art     ( 6086):   at void android.webkit.WebView.<init>(android.content.Context, android.util.AttributeSet, int) (WebView.java:555)
I/art     ( 6086):   at void android.webkit.WebView.<init>(android.content.Context, android.util.AttributeSet) (WebView.java:542)
I/art     ( 6086):   at void android.webkit.WebView.<init>(android.content.Context) (WebView.java:532)
I/art     ( 6086):   at void com.google.android.gms.internal.ads.zzbck.<init>(com.google.android.gms.internal.ads.zzbdk, com.google.android.gms.internal.ads.zzbdj, java.lang.String, boolean, boolean, com.google.android.gms.internal.ads.zzdf, com.google.android.gms.internal.ads.zzaxl, com.google.android.gms.internal.ads.zzaab, com.google.android.gms.ads.internal.zzi, com.google.android.gms.ads.internal.zza, com.google.android.gms.internal.ads.zzsd, com.google.android.gms.internal.ads.zzrf, boolean) (com.google.android.gms:play-services-ads@@18.2.0:3)
I/art     ( 6086):   at com.google.android.gms.internal.ads.zzbck com.google.android.gms.internal.ads.zzbck.zzb(android.content.Context, com.google.android.gms.internal.ads.zzbdj, java.lang.String, boolean, boolean, com.google.android.gms.internal.ads.zzdf, com.google.android.gms.internal.ads.zzaxl, com.google.android.gms.internal.ads.zzaab, com.google.android.gms.ads.internal.zzi, com.google.android.gms.ads.internal.zza, com.google.android.gms.internal.ads.zzsd, com.google.android.gms.internal.ads.zzrf, boolean) (com.google.android.gms:play-services-ads@@18.2.0:2)
I/art     ( 6086):   at java.lang.Object com.google.android.gms.internal.ads.zzbcd.call() (com.google.android.gms:play-services-ads@@18.2.0:3)
I/art     ( 6086):   at java.lang.Object com.google.android.gms.internal.ads.zzawq.zza(java.util.concurrent.Callable) (com.google.android.gms:play-services-ads-lite@@18.2.0:23)
I/art     ( 6086):   at com.google.android.gms.internal.ads.zzbbw com.google.android.gms.internal.ads.zzbcb.zza(android.content.Context, com.google.android.gms.internal.ads.zzbdj, java.lang.String, boolean, boolean, com.google.android.gms.internal.ads.zzdf, com.google.android.gms.internal.ads.zzaxl, com.google.android.gms.internal.ads.zzaab, com.google.android.gms.ads.internal.zzi, com.google.android.gms.ads.internal.zza, com.google.android.gms.internal.ads.zzsd, com.google.android.gms.internal.ads.zzrf, boolean) (com.google.android.gms:play-services-ads@@18.2.0:10)
I/art     ( 6086):   at void com.google.android.gms.internal.ads.zzahc.<init>(android.content.Context, com.google.android.gms.internal.ads.zzaxl, com.google.android.gms.internal.ads.zzdf, com.google.android.gms.ads.internal.zza) (com.google.android.gms:play-services-ads@@18.2.0:7)
I/art     ( 6086):   at void com.google.android.gms.internal.ads.zzahn.zza(com.google.android.gms.internal.ads.zzdf, com.google.android.gms.internal.ads.zzaie) (com.google.android.gms:play-services-ads@@18.2.0:54)
I/art     ( 6086):   at void com.google.android.gms.internal.ads.zzahm.run() (com.google.android.gms:play-services-ads@@18.2.0:-1)
I/art     ( 6086):   at void android.os.Handler.handleCallback(android.os.Message) (Handler.java:751)
I/art     ( 6086):   at void android.os.Handler.dispatchMessage(android.os.Message) (Handler.java:95)
I/art     ( 6086):   at void com.google.android.gms.internal.ads.zzdac.zzb(android.os.Message) (com.google.android.gms:play-services-ads-lite@@18.2.0:7)
I/art     ( 6086):   at void com.google.android.gms.internal.ads.zzauf.zzb(android.os.Message) (com.google.android.gms:play-services-ads@@18.2.0:8)
I/art     ( 6086):   at void com.google.android.gms.internal.ads.zzdac.dispatchMessage(android.os.Message) (com.google.android.gms:play-services-ads-lite@@18.2.0:5)
I/art     ( 6086):   at void android.os.Looper.loop() (Looper.java:154)
I/art     ( 6086):   at void android.app.ActivityThread.main(java.lang.String[]) (ActivityThread.java:6780)
I/art     ( 6086):   at java.lang.Object java.lang.reflect.Method.invoke!(java.lang.Object, java.lang.Object[]) (Method.java:-2)
I/art     ( 6086):   at void com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run() (ZygoteInit.java:1496)
I/art     ( 6086):   at void com.android.internal.os.ZygoteInit.main(java.lang.String[]) (ZygoteInit.java:1386)
I/art     ( 6086): 
I/art     ( 6086): Rejecting re-init on previously-failed class java.lang.Class<hk>: java.lang.NoClassDefFoundError: Failed resolution of: Landroid/webkit/RenderProcessGoneDetail;
I/art     ( 6086):   at In com.android.webview.chromium.WebViewChromiumFactoryProvider.a(android.webkit.WebView, android.content.Context) (PG:9)
I/art     ( 6086):   at void com.android.webview.chromium.WebViewChromium.init(java.util.Map, boolean) (PG:23)
I/art     ( 6086):   at void android.webkit.WebView.<init>(android.content.Context, android.util.AttributeSet, int, int, java.util.Map, boolean) (WebView.java:636)
I/art     ( 6086):   at void android.webkit.WebView.<init>(android.content.Context, android.util.AttributeSet, int, int) (WebView.java:572)
I/art     ( 6086):   at void android.webkit.WebView.<init>(android.content.Context, android.util.AttributeSet, int) (WebView.java:555)
I/art     ( 6086):   at void android.webkit.WebView.<init>(android.content.Context, android.util.AttributeSet) (WebView.java:542)
I/art     ( 6086):   at void android.webkit.WebView.<init>(android.content.Context) (WebView.java:532)
I/art     ( 6086):   at void com.google.android.gms.internal.ads.zzbck.<init>(com.google.android.gms.internal.ads.zzbdk, com.google.android.gms.internal.ads.zzbdj, java.lang.String, boolean, boolean, com.google.android.gms.internal.ads.zzdf, com.google.android.gms.internal.ads.zzaxl, com.google.android.gms.internal.ads.zzaab, com.google.android.gms.ads.internal.zzi, com.google.android.gms.ads.internal.zza, com.google.android.gms.internal.ads.zzsd, com.google.android.gms.internal.ads.zzrf, boolean) (com.google.android.gms:play-services-ads@@18.2.0:3)
I/art     ( 6086):   at com.google.android.gms.internal.ads.zzbck com.google.android.gms.internal.ads.zzbck.zzb(android.content.Context, com.google.android.gms.internal.ads.zzbdj, java.lang.String, boolean, boolean, com.google.android.gms.internal.ads.zzdf, com.google.android.gms.internal.ads.zzaxl, com.google.android.gms.internal.ads.zzaab, com.google.android.gms.ads.internal.zzi, com.google.android.gms.ads.internal.zza, com.google.android.gms.internal.ads.zzsd, com.google.android.gms.internal.ads.zzrf, boolean) (com.google.android.gms:play-services-ads@@18.2.0:2)
I/art     ( 6086):   at java.lang.Object com.google.android.gms.internal.ads.zzbcd.call() (com.google.android.gms:play-services-ads@@18.2.0:3)
I/art     ( 6086):   at java.lang.Object com.google.android.gms.internal.ads.zzawq.zza(java.util.concurrent.Callable) (com.google.android.gms:play-services-ads-lite@@18.2.0:23)
I/art     ( 6086):   at com.google.android.gms.internal.ads.zzbbw com.google.android.gms.internal.ads.zzbcb.zza(android.content.Context, com.google.android.gms.internal.ads.zzbdj, java.lang.String, boolean, boolean, com.google.android.gms.internal.ads.zzdf, com.google.android.gms.internal.ads.zzaxl, com.google.android.gms.internal.ads.zzaab, com.google.android.gms.ads.internal.zzi, com.google.android.gms.ads.internal.zza, com.google.android.gms.internal.ads.zzsd, com.google.android.gms.internal.ads.zzrf, boolean) (com.google.android.gms:play-services-ads@@18.2.0:10)
I/art     ( 6086):   at void com.google.android.gms.internal.ads.zzahc.<init>(android.content.Context, com.google.android.gms.internal.ads.zzaxl, com.google.android.gms.internal.ads.zzdf, com.google.android.gms.ads.internal.zza) (com.google.android.gms:play-services-ads@@18.2.0:7)
I/art     ( 6086):   at void com.google.android.gms.internal.ads.zzahn.zza(com.google.android.gms.internal.ads.zzdf, com.google.android.gms.internal.ads.zzaie) (com.google.android.gms:play-services-ads@@18.2.0:54)
I/art     ( 6086):   at void com.google.android.gms.internal.ads.zzahm.run() (com.google.android.gms:play-services-ads@@18.2.0:-1)
I/art     ( 6086):   at void android.os.Handler.handleCallback(android.os.Message) (Handler.java:751)
I/art     ( 6086):   at void android.os.Handler.dispatchMessage(android.os.Message) (Handler.java:95)
I/art     ( 6086):   at void com.google.android.gms.internal.ads.zzdac.zzb(android.os.Message) (com.google.android.gms:play-services-ads-lite@@18.2.0:7)
I/art     ( 6086):   at void com.google.android.gms.internal.ads.zzauf.zzb(android.os.Message) (com.google.android.gms:play-services-ads@@18.2.0:8)
I/art     ( 6086):   at void com.google.android.gms.internal.ads.zzdac.dispatchMessage(android.os.Message) (com.google.android.gms:play-services-ads-lite@@18.2.0:5)
I/art     ( 6086):   at void android.os.Looper.loop() (Looper.java:154)
I/art     ( 6086):   at void android.app.ActivityThread.main(java.lang.String[]) (ActivityThread.java:6780)
I/art     ( 6086):   at java.lang.Object java.lang.reflect.Method.invoke!(java.lang.Object, java.lang.Object[]) (Method.java:-2)
I/art     ( 6086):   at void com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run() (ZygoteInit.java:1496)
I/art     ( 6086):   at void com.android.internal.os.ZygoteInit.main(java.lang.String[]) (ZygoteInit.java:1386)
I/art     ( 6086): Caused by: java.lang.ClassNotFoundException: Didn't find class "android.webkit.RenderProcessGoneDetail" on path: DexPathList[[zip file "/data/app/com.android.chrome-2/base.apk", zip file "/data/app/com.android.chrome-2/split_autofill_assistant.apk", zip file "/data/app/com.android.chrome-2/split_autofill_assistant.config.en.apk", zip file "/data/app/com.android.chrome-2/split_autofill_assistant.config.in.apk", zip file "/data/app/com.android.chrome-2/split_config.en.apk", zip file "/data/app/com.android.chrome-2/split_config.in.apk"],nativeLibraryDirectories=[/data/app/com.android.chrome-2/lib/arm, /data/app/com.android.chrome-2/base.apk!/lib/armeabi-v7a, /data/app/com.android.chrome-2/split_autofill_assistant.apk!/lib/armeabi-v7a, /data/app/com.android.chrome-2/split_autofill_assistant.config.en.apk!/lib/armeabi-v7a, /data/app/com.android.chrome-2/split_autofill_assistant.config.in.apk!/lib/armeabi-v7a, /data/app/com.android.chrome-2/split_config.en.apk!/lib/armeabi-v7a, /data/app/com.android.chrome-2/split
I/art     ( 6086):   at java.lang.Class dalvik.system.BaseDexClassLoader.findClass(java.lang.String) (BaseDexClassLoader.java:56)
I/art     ( 6086):   at java.lang.Class java.lang.ClassLoader.loadClass(java.lang.String, boolean) (ClassLoader.java:380)
I/art     ( 6086):   at java.lang.Class java.lang.ClassLoader.loadClass(java.lang.String) (ClassLoader.java:312)
I/art     ( 6086):   at In com.android.webview.chromium.WebViewChromiumFactoryProvider.a(android.webkit.WebView, android.content.Context) (PG:9)
I/art     ( 6086):   at void com.android.webview.chromium.WebViewChromium.init(java.util.Map, boolean) (PG:23)
I/art     ( 6086):   at void android.webkit.WebView.<init>(android.content.Context, android.util.AttributeSet, int, int, java.util.Map, boolean) (WebView.java:636)
I/art     ( 6086):   at void android.webkit.WebView.<init>(android.content.Context, android.util.AttributeSet, int, int) (WebView.java:572)
I/art     ( 6086):   at void android.webkit.WebView.<init>(android.content.Context, android.util.AttributeSet, int) (WebView.java:555)
I/art     ( 6086):   at void android.webkit.WebView.<init>(android.content.Context, android.util.AttributeSet) (WebView.java:542)
I/art     ( 6086):   at void android.webkit.WebView.<init>(android.content.Context) (WebView.java:532)
I/art     ( 6086):   at void com.google.android.gms.internal.ads.zzbck.<init>(com.google.android.gms.internal.ads.zzbdk, com.google.android.gms.internal.ads.zzbdj, java.lang.String, boolean, boolean, com.google.android.gms.internal.ads.zzdf, com.google.android.gms.internal.ads.zzaxl, com.google.android.gms.internal.ads.zzaab, com.google.android.gms.ads.internal.zzi, com.google.android.gms.ads.internal.zza, com.google.android.gms.internal.ads.zzsd, com.google.android.gms.internal.ads.zzrf, boolean) (com.google.android.gms:play-services-ads@@18.2.0:3)
I/art     ( 6086):   at com.google.android.gms.internal.ads.zzbck com.google.android.gms.internal.ads.zzbck.zzb(android.content.Context, com.google.android.gms.internal.ads.zzbdj, java.lang.String, boolean, boolean, com.google.android.gms.internal.ads.zzdf, com.google.android.gms.internal.ads.zzaxl, com.google.android.gms.internal.ads.zzaab, com.google.android.gms.ads.internal.zzi, com.google.android.gms.ads.internal.zza, com.google.android.gms.internal.ads.zzsd, com.google.android.gms.internal.ads.zzrf, boolean) (com.google.android.gms:play-services-ads@@18.2.0:2)
I/art     ( 6086):   at java.lang.Object com.google.android.gms.internal.ads.zzbcd.call() (com.google.android.gms:play-services-ads@@18.2.0:3)
I/art     ( 6086):   at java.lang.Object com.google.android.gms.internal.ads.zzawq.zza(java.util.concurrent.Callable) (com.google.android.gms:play-services-ads-lite@@18.2.0:23)
I/art     ( 6086):   at com.google.android.gms.internal.ads.zzbbw com.google.android.gms.internal.ads.zzbcb.zza(android.content.Context, com.google.android.gms.internal.ads.zzbdj, java.lang.String, boolean, boolean, com.google.android.gms.internal.ads.zzdf, com.google.android.gms.internal.ads.zzaxl, com.google.android.gms.internal.ads.zzaab, com.google.android.gms.ads.internal.zzi, com.google.android.gms.ads.internal.zza, com.google.android.gms.internal.ads.zzsd, com.google.android.gms.internal.ads.zzrf, boolean) (com.google.android.gms:play-services-ads@@18.2.0:10)
I/art     ( 6086):   at void com.google.android.gms.internal.ads.zzahc.<init>(android.content.Context, com.google.android.gms.internal.ads.zzaxl, com.google.android.gms.internal.ads.zzdf, com.google.android.gms.ads.internal.zza) (com.google.android.gms:play-services-ads@@18.2.0:7)
I/art     ( 6086):   at void com.google.android.gms.internal.ads.zzahn.zza(com.google.android.gms.internal.ads.zzdf, com.google.android.gms.internal.ads.zzaie) (com.google.android.gms:play-services-ads@@18.2.0:54)
I/art     ( 6086):   at void com.google.android.gms.internal.ads.zzahm.run() (com.google.android.gms:play-services-ads@@18.2.0:-1)
I/art     ( 6086):   at void android.os.Handler.handleCallback(android.os.Message) (Handler.java:751)
I/art     ( 6086):   at void android.os.Handler.dispatchMessage(android.os.Message) (Handler.java:95)
I/art     ( 6086):   at void com.google.android.gms.internal.ads.zzdac.zzb(android.os.Message) (com.google.android.gms:play-services-ads-lite@@18.2.0:7)
I/art     ( 6086):   at void com.google.android.gms.internal.ads.zzauf.zzb(android.os.Message) (com.google.android.gms:play-services-ads@@18.2.0:8)
I/art     ( 6086):   at void com.google.android.gms.internal.ads.zzdac.dispatchMessage(android.os.Message) (com.google.android.gms:play-services-ads-lite@@18.2.0:5)
I/art     ( 6086):   at void android.os.Looper.loop() (Looper.java:154)
I/art     ( 6086):   at void android.app.ActivityThread.main(java.lang.String[]) (ActivityThread.java:6780)
I/art     ( 6086):   at java.lang.Object java.lang.reflect.Method.invoke!(java.lang.Object, java.lang.Object[]) (Method.java:-2)
I/art     ( 6086):   at void com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run() (ZygoteInit.java:1496)
I/art     ( 6086):   at void com.android.internal.os.ZygoteInit.main(java.lang.String[]) (ZygoteInit.java:1386)
I/art     ( 6086): 
D/ConnectivityManager( 6086): requestNetwork; CallingUid : 10647, CallingPid : 6086
E/libEGL  ( 6086): validate_display:99 error 3008 (EGL_BAD_DISPLAY)
D/ConnectivityManager( 6086): requestNetwork; CallingUid : 10647, CallingPid : 6086
W/AudioCapabilities( 6086): Unsupported mime audio/mpeg-L1
W/AudioCapabilities( 6086): Unsupported mime audio/mpeg-L2
W/AudioCapabilities( 6086): Unsupported mime audio/x-ms-wma
W/AudioCapabilities( 6086): Unsupported mime audio/x-ima
W/VideoCapabilities( 6086): Unrecognized profile/level 1/32 for video/mp4v-es
W/VideoCapabilities( 6086): Unrecognized profile/level 32768/2 for video/mp4v-es
W/VideoCapabilities( 6086): Unrecognized profile/level 32768/64 for video/mp4v-es
W/VideoCapabilities( 6086): Unsupported mime video/wvc1
W/VideoCapabilities( 6086): Unsupported mime video/x-ms-wmv
W/VideoCapabilities( 6086): Unrecognized profile/level 1/32 for video/mp4v-es
W/VideoCapabilities( 6086): Unrecognized profile/level 32768/2 for video/mp4v-es
W/VideoCapabilities( 6086): Unrecognized profile/level 32768/64 for video/mp4v-es
W/VideoCapabilities( 6086): Unsupported mime video/wvc1
W/VideoCapabilities( 6086): Unsupported mime video/x-ms-wmv
W/VideoCapabilities( 6086): Unsupported mime video/x-ms-wmv7
W/VideoCapabilities( 6086): Unsupported mime video/x-ms-wmv8
W/VideoCapabilities( 6086): Unsupported mime video/mp43
W/VideoCapabilities( 6086): Unrecognized profile 2130706433 for video/avc
W/VideoCapabilities( 6086): Unrecognized profile/level 1/32 for video/mp4v-es
W/VideoCapabilities( 6086): Unrecognized profile/level 32768/2 for video/mp4v-es
W/VideoCapabilities( 6086): Unrecognized profile/level 32768/64 for video/mp4v-es
I/VideoCapabilities( 6086): Unsupported profile 4 for video/mp4v-es
W/VideoCapabilities( 6086): Unsupported mime video/sorenson
W/flutter ( 6086): onRewardedVideoAdFailedToLoad: 3
I/Ads     ( 6086): Ad failed to load : 3
Application finished.
```

**Comment by zukinaru** [comment]
This case is solved, this is my fault because of the modified version of `main.java` where i manually import library version generated by flutter, but forgot to add `native_ads` library.

However, I face new errors, please see new issue
